protobuf/transform.go

```diff
@@ -225,19 +250,32 @@ func (t *Transformer) transformField(pkg *Package, msg *Message, field *scanner.
 	return f
 }
 
-func defaultOptionsForStructField(field *scanner.Field) Options {
+func defaultOptionsForStructField(t *Transformer, field *scanner.Field) Options {
```

Comment:
is `defaultOptionsForStructField` used anywhere else? If not, we should consider making this a method of `Transformer`. That way we would not need to pass it around